src/registries/uniswapv2.ts

```diff
@@ -26,7 +26,7 @@ export class RegistryUniswapV2 extends Registry {
 
 	findPairs = async (tokenWhitelist: Address[]): Promise<Pair[]> =>  {
 		let pairsFetched
-		if (!this.opts?.fetchUsingAllPairs) {
+		if (this.opts?.fetchUsingAllPairs === false) {
```

Comment:
unless `fetchUsingAllPairs` is not `undefined` and is explicitly set to `false`
we will use using the `allPairsLength` and `allPairs` factory methods